health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MapFilterIndex.js

```
@@ -0,0 +1,41 @@
+import React,{Fragment} from "react";
+import * as DigitSvgs from "@egovernments/digit-ui-svg-components";
+
+const IconCollection = {...DigitSvgs };
+
+export const FilterItemBuilder = ({ item, MapFilters, t }) => {
+    let temp = MapFilters?.find((e) => e?.name === item)?.icon?.index;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'temp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)